packages/x-charts/tsconfig.json

```diff
@@ -10,5 +10,5 @@
     ],
     "skipLibCheck": true
   },
-  "include": ["src/**/*", "../../test/utils/addChaiAssertions.ts", "../../test/utils/init.ts"]
```

Comment:
Is there a reason for removing this file?

Reply:
it is useless 😆

Reply:
We are pretty much including the types here, but `../../test/utils/addChaiAssertions.ts ` already add the types necessary